rust/Cargo.toml.in

```diff
@@ -29,12 +29,16 @@ num-derive = "0.2"
 num-traits = "0.2"
 widestring = "0.4"
 
-der-parser = "3.0"
-kerberos-parser = "0.4"
+der-parser = "4.0"
+kerberos-parser = "0.5"
 
 ntp-parser = "0.4"
 ipsec-parser = "0.5"
-snmp-parser = "0.5"
+snmp-parser = "0.6"
 tls-parser = "0.9"
 x509-parser = "0.6.5"
 libc = "0.2.67"
+
+[dev-dependencies]
+hex-literal = "0.2"
+test-case = "1.0"
```

Comment:
Can you tell us why you chose this test helper? We're not opposed to it, but are cautious with pulling in new dependencies, and in particular how aggressive they may be at dropping support for older versions of Rust.

Reply:
Right now I have 2 concerns with the test-case crate.  Cargo test fails, and their own CI tests in Github also fail.

Reply:
I chose it in order to introduce parameterized testing, this allows to easily add more tests and to reduce code duplication.

There's no particular reason why this crate over another. I've used both `test-case` and `rstest` in the past. I'm not oppose to changing it either.